.gitignore

```diff
@@ -0,0 +1,340 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
```

Comment:
I usually try to avoid committing editor specific files to repositories, that's what a global `.gitignore` file is for.

Reply:
This was auto-generated for me, I'll keep it for now!

Reply:
No, it's fine 🙂